manifests/plugin/aggregation.pp

```diff
@@ -1,8 +1,8 @@
 # collectd::plugin::aggregation
 class collectd::plugin::aggregation (
-  Enum['present', 'absent'] $ensure = 'present',
-  Optional[Integer[1]] $interval    = undef,
   Hash $aggregators                 = {},
+  Enum['present', 'absent'] $ensure = 'present',
```

Comment:
I believe there's another lint check that prefers ensure to be the first param. Perhaps that's only for instantiation though.

Reply:
correct, that's only for resources (and as far as I know thats active in our setup)